tests/ordering/tests.py

```diff
@@ -92,45 +92,49 @@ def test_order_by_nulls_first_and_last(self):
         with self.assertRaisesMessage(ValueError, msg):
             Article.objects.order_by(F("author").desc(nulls_last=True, nulls_first=True))
 
+    def assertQuerysetEqualReversible(self, queryset, sequence):
+        self.assertSequenceEqual(queryset, sequence)
+        self.assertSequenceEqual(queryset.reverse(), list(reversed(sequence)))
+
     def test_order_by_nulls_last(self):
         Article.objects.filter(headline="Article 3").update(author=self.author_1)
         Article.objects.filter(headline="Article 4").update(author=self.author_2)
         # asc and desc are chainable with nulls_last.
-        self.assertSequenceEqual(
-            Article.objects.order_by(F("author").desc(nulls_last=True)),
+        self.assertQuerysetEqualReversible(
+            Article.objects.order_by(F("author").desc(nulls_last=True), 'headline'),
```

Comment:
the tests were failing because the ordering wasn't fully specified in the SQL query, causing the test to rely on undefined behaviour from the DB. This fixes that.